website/source/docs/commands/acl/acl-role.html.md.erb

```diff
@@ -0,0 +1,315 @@
+  unique ID prefix but will error if the prefix matches multiple role IDs
+
+* `-meta` - Indicates that role metadata such as the content hash and raft
+  indices should be shown for each entry
+
+* `-name=<string>` - The role name.
+
+* `-no-merge` - Do not merge the current role information with what is provided
+  to the command. Instead overwrite all fields with the exception of the role
+  ID which is immutable.
+
+* `-policy-id=<value>` - ID of a policy to use for this role. May be specified
+  multiple times
+
+* `-policy-name=<value>` - Name of a policy to use for this role. May be
+  specified multiple times
+
+* `-service-identity=<value>` - Name of a service identity to use for this
+  role. May be specified multiple times. Format is the `SERVICENAME` or
+  `SERVICENAME:DATACENTER1,DATACENTER2,...`
+
+### Examples
+
+Update a role:
+
+```sh
+$ consul acl role update -id 57147d87-6bf7-f794-1a6e-7d038c4e4ae9 \
+    -description 'better description' -service-identity 'crawler'
+Role updated successfully
+ID:           57147d87-6bf7-f794-1a6e-7d038c4e4ae9
+Name:         crawler
+Description:  better description
+Policies:
+   2f8f99c7-edd9-2f09-7e4b-a1f519eb4fc2 - crawler-kv
+Service Identities:
+   crawler (Datacenters: all)
+```
+
+Rename a role by prefix:
+
+```sh
+$ consul acl role update -id 57147 -name web-crawler
+Role updated successfully
+ID:           57147d87-6bf7-f794-1a6e-7d038c4e4ae9
+Name:         web-crawler
+Description:  better description
+Policies:
+   2f8f99c7-edd9-2f09-7e4b-a1f519eb4fc2 - crawler-kv
+Service Identities:
+   crawler (Datacenters: all)
+```
+
+## `delete`
+
+Command: `consul acl role delete`
+
+This command deletes a role. Roles may be deleted by their ID or by name.
+
+### Usage
+
+Usage: `consul acl role delete [options]`
+
+#### Options
+
+* [Common Subcommand Options](#common-subcommand-options)
+
+* `-id=<string>` - The ID of the role to delete. It may be specified as a
+  unique ID prefix but will error if the prefix matches multiple role IDs.
+
+* `-name=<string>` - The Name of the role to delete.
+
+### Examples
+
+Delete a role by prefix:
+
+```sh
+$ consul acl role delete -id 57147
+Role "57147d87-6bf7-f794-1a6e-7d038c4e4ae9" deleted successfully
+```
+
+Delete a role by name:
+
+```sh
+$ consul acl role delete -name crawler
+Role "a365fdc9-ac71-e754-0645-7ab6bd747301" deleted successfully
+```
+
+## `list`
+
+Command: `consul acl role list`
+
+This command lists all roles. By default it will not show metadata.
+
+### Usage
+
+Usage: `consul acl role list`
+
+#### Options
+
+* [Common Subcommand Options](#common-subcommand-options)
+
+* `-meta` - Indicates that role metadata such as the content hash and
+   Raft indices should be shown for each entry.
+
+### Examples
+
+Default listing.
+
+```sh
+$ consul acl role list
+web-crawler:
+   ID:           57147d87-6bf7-f794-1a6e-7d038c4e4ae9
+   Description:  better description
```

Comment:
kait: can we add a real description forthese or just put "A description of the role"